tox.ini

```diff
@@ -18,8 +18,8 @@ deps =
     online: pytest-rerunfailures
     online: pytest-timeout
 commands =
-    offline,astropydev,numpydev: pytest --verbose --pyargs sunpy -m "not figure" --cov=sunpy --rootdir={toxinidir} {posargs} {toxinidir}/docs
-    online: pytest --verbose --pyargs sunpy --timeout=300 -m "not figure" --remote-data=any --cov=sunpy  --rootdir={toxinidir} {posargs} {toxinidir}/docs
+    offline,astropydev,numpydev: pytest --verbose --pyargs sunpy -m "not figure" --cov=sunpy --rootdir={toxinidir} --cov-config={toxinidir}/setup.cfg {posargs} {toxinidir}/docs
+    online: pytest --verbose --pyargs sunpy --timeout=300 -m "not figure" --remote-data=any --cov=sunpy --rootdir={toxinidir} --cov-config={toxinidir}/setup.cfg {posargs} {toxinidir}/docs
```

Comment:
I need to figure out some way to define a base command. These are getting ugly and out of hand.